mern/client/src/withRouter.js

```diff
@@ -0,0 +1,16 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
This should be organized into a different folder to make the code cleaner. Maybe not just in SRC, put it in the generic buttons folder or even the utils folder.